src/traccuracy/loaders/_point.py

```diff
@@ -0,0 +1,90 @@
+from typing import Optional, cast
+
+import networkx as nx
+import pandas as pd
+
+from traccuracy._tracking_graph import TrackingGraph
+
+
+def load_point_data(
+    path: str | None = None,
+    df: Optional[pd.DataFrame] = None,  # noqa: UP007, bar syntax breaks docks build
+    parent_column: str = "parent",
+    id_column: str | None = None,
+    pos_columns: tuple[str, ...] = ("z", "y", "x"),
+    time_column: str = "t",
+    name: str | None = None,
+    sep: str | None = None,
+) -> TrackingGraph:
+    """Load point-based tracking data into a TrackingGraph from a csv-like file
```

Comment:
Add to the docs that any additional columns are added as node attributes (they are, right?)

Reply:
I decided not to add any additional columns since traccuracy wouldn't use the, but I could add a flag in to make it optional in case someone wants it for visualization downstream.

Reply:
I think it makes sense to ignore them if traccuracy doesn't use them - if people really want them they can make a custom loader.